### DISCUSSION TITLE
這裡也有人看vtuber嗎

### DISCUSSION THREAD

**Discussion by eric202501** [1101-GS4719-C/Python-Community-GS4719-C#1650] [閒聊八卦]
我原本是個單純的小男孩，但在看過vtuber之後就回不去了，現在是fauna的狗了，有人要來推廣一下自己的推嗎

**Comment by davidteng00000** [comment]
現在已經有完全AI操控的Vtuber了，我認為透過技術達成自動化將會使這個產業更具獲利能力

**Comment by mrweixd** [comment]
我覺得EN同屆mumei也不錯，可愛的貓頭鷹化解人心，有時還會有與之不同的反差萌，十分推薦！https://www.youtube.com/@NanashiMumei

**Comment by v0ouo** [comment]
看vtuber的都宅斃了(沒，對於我來說，一直看他們直播會花太多時間，所以頂多就看別人剪的精華而已，另外我也沒有花任何錢在這方面上，畢竟花在一個虛擬的東西上，好像有點太浪費了，還不如拿去吃飯。
